packages/components/src/FilterDatePicker/_docs/FilterDatePicker.stories.tsx

```diff
@@ -0,0 +1,355 @@
+import React, { useEffect, useState } from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import Highlight from "react-highlight"
+import { Paragraph } from "@kaizen/typography"
+import { renderTriggerControls } from "~components/Filter/_docs/controls/renderTriggerControls"
+import {
+  FilterButton,
+  FilterButtonProps,
+  FilterButtonRemovable,
+} from "~components/FilterButton"
+import { ComponentDocsTemplate } from "../../../../../storybook/components/DocsContainer"
+import {
+  DateValidationResponse,
+  FilterDatePicker,
+  ValidationMessage,
+} from "../index"
+import { FilterDatePickerField } from "../subcomponents/FilterDatePickerField"
+import { defaultMonthControls } from "./controls/defaultMonthControls"
+import { disabledDaysControls } from "./controls/disabledDaysControls"
+import { validationControls } from "./controls/validationControls"
+
+export default {
+  tags: ["autodocs"],
```

Comment:
The popover has been fixed to use "absolute" instead of "fixed" so your docs will need to be updated to use mdx and `NoClipCanvas` :P